velox/exec/tests/HashJoinBridgeTest.cpp

```diff
@@ -574,4 +574,19 @@ VELOX_INSTANTIATE_TEST_SUITE_P(
     HashJoinBridgeTest,
     testing::ValuesIn(HashJoinBridgeTest::getTestParams()));
 
+TEST(HashJoinBridgeTest, getHashJoinTableSpillType) {
+  const RowTypePtr tableType = ROW({"k1", "k2"}, {BIGINT(), BIGINT()});
+  const RowTypePtr spillType = hashJoinTableSpillType(tableType);
+  ASSERT_EQ(tableType->size() + 1, spillType->size());
```

Comment:
Can we create an expected RowType and assert that expected and actual are the same using operator== ?